skimage/segmentation/_clear_border.py

```diff
@@ -5,12 +5,10 @@
 def clear_border(labels, buffer_size=0, bgval=0, in_place=False):
     """Clear objects connected to the label image border.
 
-    The changes will be applied directly to the input.
-
     Parameters
     ----------
-    labels : (N, M) array of int
-        Label or binary image.
+    labels : (M[, N[, ..., P]]) array of int
+        Label or binary image
```

Comment:
@soupault - I didn't change the phrasing of that one.. can change it to `Imaging data labels` if that's preferred though.